py/parse_and_notify.py

```diff
@@ -218,15 +305,18 @@ def main():
             f"{dry_run}Topicctl ran apply on topic {topic_content.name} "
             f"in region {SENTRY_REGION}"
         )
-        text = topic_content.render_table()
-        if len(text) > 3950:
-            text = (
-                "Changes exceed 4000 character limit, "
+        dd_table = topic_content.render_table()
+        slack_table = topic_content.render_slack_msg()
+        if len(dd_table) > 3950 or len(slack_table) > 2950:
```

Comment:
What are those conditions for ?

Reply:
DD max event length is 4000, slack max message length is 3000. I set them slightly lower here to allow some wiggle room.
I'll add a comment

Reply:
Please make them constant and avoid magic numbers in code